ras_party/views/account_view.py

```diff
@@ -113,6 +113,21 @@ def change_respondent_enrolment_status():
     return make_response(jsonify('OK'), 200)
 
 
+@account_view.route('/respondents/disable-user-enrolments', methods=['PUT'])
```

Comment:
I think this should be a POST.  A put usually contains everything that is going to replace what is there.  Here, we're just providing an email and saying 'go change it for us'

Reply:
Disagree , if anything it should be a PATCH